### PR TITLE
Update Jet Set

### PR DESCRIPTION
**Pull request by cbirdsong**
Need to update the @import and font URLs. (Had to rehost on Netlify)

### CODE REVIEW

**Comment by cbirdsong** [comment]
Oops, did it wrong. 🤦‍♂️

If you regenerate the preview it should have a hard-coded URL for fonts again.

**Comment by 5t3ph** [comment]
@cbirdsong - hmm, rebuilt and still has relative font links:

https://deploy-preview-88--moderncss-styles.netlify.app/styles/jet-set/

**Comment by cbirdsong** [comment]
Hmm, going [straight to the sheet](https://stylestage-jet-set.birdsong.dev/css/style.css) shows them as absolute. Looks like CORS errors in the console – should I remove the my domain and have it point straight at the Netlify temp domain?

**Comment by cbirdsong** [comment]
Just redid it as https://stylestage-jet-set.netlify.app, we can see if that fixes it?

File: src/_data/styles/jet-set.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "title": "Jet Set",
   "author": "Cory Birdsong",
-  "stylesheet": "https://cbirdsong.github.io/stylestage-jet-set/style.css",
+  "stylesheet": "https://stylestage-jet-set.birdsong.dev/css/style.css",
```
Comment (5t3ph) [inline src/_data/styles/jet-set.json:4]:
```suggestion
  "stylesheet": "https://stylestage-jet-set.netlify.app/css/style.css",
```

**Comment by 5t3ph** [comment]
@cbirdsong - looks like it's a CORS issue - I'm not sure if that can be resolved, you will have to look into that for Netlify. https://deploy-preview-88--moderncss-styles.netlify.app/styles/jet-set/ < check out the console

**Comment by cbirdsong** [comment]
Changed it back to the original Github Pages URL, that should be fine? Decided to just commit the compiled files directly instead of worrying about them being built by the deploy process.

**Review by 5t3ph — APPROVED** [review]
@cbirdsong works! updating now...

**Pull request merged**